server/activityItem/fetch.ts

```diff
@@ -0,0 +1,201 @@
+import { Op } from 'sequelize';
+
+import * as types from 'types';
+import {
+	ActivityAssociations,
+	ActivityAssociationType,
+	activityAssociationTypes,
+	ActivityAssociationIds,
+	ActivityItemsContext,
+	WithId,
+	IdIndex,
+} from 'types';
```

Comment:
Hmm, the approach I've been settling on is to use `types.X` when there's ambiguity with model `X` and to...sorta?...prefer importing the type directly otherwise. Happy to defer to a unified approach if you feel strongly about it tho.